tests-utils/mock-requsts.ts

```diff
@@ -1,6 +1,6 @@
 import nock from 'nock';
-
-export const createGetRequest = (host, path, headers, status, response, respHeaders) => {
+import type { DefaultHeaders } from 'fastify';
```

Comment:
typo in file name mock-requsts.ts

Reply:
I forgot this one! I Will fix It in master